libs/src/services/creatorNode/index.js

```diff
@@ -573,7 +586,6 @@ class CreatorNode {
       url: '/users/logout',
       method: 'post'
     }, false)
-    this.authToken = null
```

Comment:
why delete this?

Reply:
yeah i think we should probably keep this in

Reply:
I moved/wrapped it in the `clearBrowserSession` function which I called in both places that this was before - because I felt that was more descriptive. LMK your thoughts!